region/tourian/main/Tourian Escape Room 4.json

```diff
@@ -567,6 +551,110 @@
         "Alternatively, jump and force the pirates to shoot just before starting to climb."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump (Longer Acid Dip)",
+      "requires": [
+        {"notable": "Reverse Tourian Escape Room 4 Bootless Walljumpless Space Jump"},
+        "SpaceJump",
+        "canTrickyJump",
+        {"acidFrames": 90}
```

Comment:
should these have a leave normally exit condition, so the air runway cant be used?

Reply:
Good catch. Door unlocks may also be a problem for some of these, will take another look